aol/aol-htb-exports.js

```diff
@@ -0,0 +1,5 @@
+//? if (FEATURES.GPT_LINE_ITEMS) {
+shellInterface.AOLHtb = {
```

Comment:
The rename from `shellInterface.AolHtb` to `shellInterface.AOLHtb` and the deletion of `shellInterface.AolModule` may impact the ability of legacy creatives to find the render function. For the safest bet and consistency, please change back all `AOLHtb` to `AolHtb`, and add back `shellInterface.AolModule`.

Reply:
Done.